Remove submission field from Job model

yenaing-oo opened the pull request:
Closes https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/issues/417

danloveg reviewed (changes requested):
The `submit_buttons_bottom` block in the `submission_change_form.html` should be updated to reflect the fact that there is no `has_generated_bag` context variable now.

yenaing-oo commented:
> The `submit_buttons_bottom` block in the `submission_change_form.html` should be updated to reflect the fact that there is no `has_generated_bag` context variable now.

I've addressed that here: https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/438/commits/d1e52e272ea9ffa22321820f5c6586ad6e61e4fc

I also renamed the migration for removing the `submission` field, since migration `045` was used in the previous sub-issue https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/issues/416.

I believe both will get squashed into one at the end.

danloveg commented on app/recordtransfer/migrations/0046_remove_job_submission.py:
Sorry about the confusion with the migrations, but you'll need to merge 415-job-overhaul into this branch and then rename this migration one more time.

That should be done *after* #439 gets merged into 415-job-overhaul, since that issue also has new migrations.

yenaing-oo replied:
No worries at all, this all makes sense to me.

yenaing-oo replied:
I've fixed the migrations here, so that the latest one is now `049`: https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/438/commits/ea29d36586f460664b3a5757ca2cb238ed3692e3

danloveg replied:
Perfect

danloveg reviewed (changes requested):
One more change, but this should be done after #439 is merged into the target branch. The migrations are all dependent on each other which makes the ordering a bit tedious to manage.

danloveg commented:
> I believe both will get squashed into one at the end.

Yes, we'll squash these once all the migrations are made.

danloveg reviewed (approved):
This must be a record for most closed pull requests in an hour. Looks good!
